src/ViewLayer/RaceModeDisplay/RaceModeDashboardView.h

```diff
@@ -3,6 +3,8 @@
 #include <QObject>
 #include <QLabel>
 #include <QScopedPointer>
+#include <QPropertyAnimation>
```

Comment:
These can be forward declared here and included in the cpp because the implementation of these classes is only needed in the cpp file